src/visuals/sprite.rs

```diff
@@ -123,6 +146,24 @@ impl Sprite {
         args.into()
     }
 
+    pub fn new_animation<T>(
```

Comment:
Add comment from following commit in this one

You can edit this commit using `$ git commit --amend`.